packages/ckeditor5-utils/src/dom/rect.ts

```diff
@@ -576,7 +576,7 @@ function getElementOverflow( element: HTMLElement | Range ): string {
  * (and scrollable) ancestors.
  */
 function shiftRectToCompensatePositionedAncestor( rect: Rect, positionedElementAncestor: HTMLElement ): void {
-	const ancestorPosition = new Rect( positionedElementAncestor );
```

Comment:
This change looks like it's not related and as we previously debugged it seems invalid.